packages/core-mobile/index.js

```diff
@@ -35,9 +38,35 @@ if (DevDebuggingConfig.STORYBOOK_ENABLED) {
 }
 
 AppRegistry.registerComponent(appName, () => AppEntryPoint)
+FCMService.listenForMessagesBackground()
```

Comment:
nit: create a FirebaseService to listenForMessagesBackground and configure?

Reply:
Is this what you meant?
[Move initialisation of AppCheck to AppCheckService](https://github.com/ava-labs/core-mobile/pull/1659/commits/e4308667da340909a029cb02d7cdae8280a42065)